frontend/src/app/page.tsx

```diff
@@ -16,7 +16,7 @@ export default function App() {
     // These are example parameters. Replace with your actual values
     const params = new URLSearchParams({
       client_id: process.env.NEXT_PUBLIC_GOOGLE_AUTH_CLIENT_ID!,
-      redirect_uri: "http://localhost:8080/api/auth/callback/google",
+      redirect_uri: `${window.location.origin}/api/auth/callback/google`,
```

Comment:
I don't want to make it a blocker, if this work, this is fine with me.
But since next-auth is setup, why not simply using signIn('google') from next-auth when clicking on the google signin button?

Reply:
Right, much cleaner solution